PhysicsTools/HeppyCore/scripts/heppy_batch.py

```diff
@@ -211,7 +211,7 @@ def batchScriptIC(jobDir):
 python {cmssw}/src/PhysicsTools/HeppyCore/python/framework/looper.py pycfg.py config.pck
 echo
 echo 'sending the job directory back'
-mv Loop/* ./
+cp -r Loop/* ./ && rm -r Loop
```

Comment:
The "Loop" directory still existed, which meant when resubmitting "Loop_1" would be made instead. I can change this to a mv then rm instead of a cp and rm.